tests/ios/app/BUILD.bazel

```diff
@@ -280,3 +281,12 @@ ios_application(
         ":SwiftLib",
     ],
 )
+
+genrule(
+    name = "TopLevelBundles",
+    cmd = "file $(SRCS) && touch $(OUTS)",
```

Comment:
Will `file` still succeed even if $SRCS don't exist? On my machine, I think `file` returns 0, even if i pass it nonsense filenames.

How does this genrule test the changes in this PR? That's not clear to me. Thanks!

Reply:
This isn't testing anything inside of the PR and `file` is added there mainly to get the `$(SRCS)` on the CLI invocation. It exercises the redundant build path with a single target - we could add assertion on it with #186.

Reply:
please add a comment for the target explaining what it's exercising?

Reply:
+1 also spent some time trying to understand the need for this `genrule`

Reply:
For now, maybe it's easier to just add this along with a test case 🙌 , I'm going to look more into using an aspect in addition to an analysis test to see if we can get an assertion on this